digestbot/core/DBModels/Message/Message.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+
+@dataclass
+class Message:
+    id: int
+    username: str
+    text: str
+    date: datetime
```

Comment:
```suggestion
    date: Decimal
```
I have similar dataclass now, we should deduplicate it after merge.
What about putting all dataclasses to separate module?

Reply:
If you commit your changes earlier than me - I can pull your changes into my branch. 
But dataclasses - it's about data from out project. Isn't it?